ColorPickerPOC/SpectrumIndicator.js

```diff
@@ -36,7 +39,22 @@ const CircleIndicator = ({bgColor, canvasRef, isIndicatorActive, selectedColorHa
 			const hexColor = getHexColorFromGradient(canvasRef, x, y);
 			setIndicatorBgColor(hexColor);
 		}
-	}, [canvasRef, x, y, isIndicatorActive, setIsIndicatorActive, setIndicatorBgColor]);
+
+		if(isIndicatorActive){
+			if(holding){
```

Comment:
Please leave empty space between if and parenthesis. After closing parenthesis as well.